src/odin/ingestion/afc/afc_archive.py

```diff
@@ -143,56 +135,46 @@ def setup_job(self) -> None:
         else:
             raise IndexError(f"{self.table} not found in 'tableinfos' API endpoint.")
 
-        # set self.last_sid
-        self.last_sid = None
+        # set self.pq_sid from parquet dataset
+        self.pq_sid: int | None = None
+        self.max_sid: int | None = None
```

Comment:
`self.max_sid` only purpose is to determine if job will need immediate re-run, or not.